event_sale_reservation/views/product_template_view.xml

```diff
@@ -6,17 +6,18 @@
         <field name="name">Allow products that are event registrations</field>
         <field name="model">product.template</field>
         <field name="groups_id" eval="[(4, ref('event.group_event_user'))]" />
-        <field name="inherit_id" ref="event_sale.event_sale_product_template_form" />
+        <field name="inherit_id" ref="product.product_template_form_view" />
         <field name="arch" type="xml">
-            <field name="event_ok" position="attributes">
+            <field name="detailed_type" position="attributes">
```

Comment:
Seeing this, for me it seems what you have to do is to add a new `detailed_type`: `event_reservation`, instead of all this stuff.